### DISCUSSION TITLE
Add an option to set default link type in link field

### DISCUSSION THREAD

**Discussion by droemmelbert** [craftcms/cms#15520] [Ideas] [answered]
It would be great to set a default link type in the new link field.

For example:

I create a link field for a customer allowing to link to URL, assets & entries. This field will _usually_ be used to link to **entries**. However, every now and then the customer might also link to a URL or asset. I would like to set the default link type to **entry** in order to make it easier and more intuitive on how the customer is going to fill in this link field.

**Comment by mmikkel** [comment, marked as answer]
Agree! I suggested the same thing here: https://github.com/craftcms/cms/discussions/15516

**Reply by droemmelbert** [reply]
Couldn't find anything earlier. Thanks for the link!
